tic-tac-toe/src/app/components/board/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Square from '../square';
+
+class Board extends React.Component {
```

Comment:
We import Component from react and we you it directly here. 

import React, { Component }  from 'react';